app/containers/CreateImportWallet/ImportWallet/ImportWallet.jsx

```diff
@@ -0,0 +1,139 @@
+// @flow
+import React from 'react'
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
+import { ROUTES } from '../../../core/constants'
+import CloseButton from '../../../components/CloseButton'
+import BackButton from '../../../components/BackButton'
+import Button from '../../../components/Button'
+import CreateImportWalletForm from '../../../components/CreateImportWalletForm'
+import FullHeightPanel from '../../../components/Panel/FullHeightPanel'
+import { loadWalletRecovery } from '../../Settings/Settings'
+import ImportIcon from '../../../assets/icons/import.svg'
+import AddIcon from '../../../assets/icons/add.svg'
+import styles from './ImportWallet.scss'
+
+type Props = {
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
I think we can get rid of these by using maybe types here https://flow.org/en/docs/types/maybe/

Reply:
If I add only one comment before the `Props` definition, it is still working:

```Javascript
// @flow
import React from 'react'
import { Tab, Tabs, TabList, TabPanel } from 'react-tabs'
import { ROUTES } from '../../../core/constants'
import CloseButton from '../../../components/CloseButton'
import BackButton from '../../../components/BackButton'
import Button from '../../../components/Button'
import CreateImportWalletForm from '../../../components/CreateImportWalletForm'
import FullHeightPanel from '../../../components/Panel/FullHeightPanel'
import { loadWalletRecovery } from '../../Settings/Settings'
import ImportIcon from '../../../assets/icons/import.svg'
import AddIcon from '../../../assets/icons/add.svg'
import styles from './ImportWallet.scss'

/* eslint-disable react/no-unused-prop-types */
type Props = {
  authenticated: boolean,
  setAccounts: (Array<Object>) => any,
  showSuccessNotification: ({ message: string }) => string,
  showErrorNotification: ({ message: string }) => string,
}

```

**Are you ok, with this proposal?**

Reply:
This is fine @dauTT 👍